packages/devtools-kit/src/_types/rpc.ts

```diff
@@ -56,7 +56,8 @@ export interface ServerFunctions {
   getImageMeta(filepath: string): Promise<ImageMeta | undefined>
   getTextAssetContent(filepath: string, limit?: number): Promise<string | undefined>
   writeStaticAssets(token: string, file: { name: string; data: string }[], path: string): Promise<string[]>
-  deleteStaticAsset(token: string, file: string): Promise<void>
+  writeStaticAsset(token: string, filepath: string, data: string): Promise<void>
```

Comment:
Can we just reuse `writeStaticAssets` and pass an array to do so?

Reply:
Sure! (I don't know why I dindn't do that at first place sorry lol) thanks